src/lib/TranscriptEditor/TimedTextEditor/WrapperBlock.module.css

```diff
@@ -1,44 +1,96 @@
 @value color-labs-red, color-light-grey, color-mid-grey, color-dark-grey from '../colours.module.css';
 
-.WrapperBlock {
-  height: 100%;
-  width: 100%;
-  margin-bottom: 2em;
-  line-height: 1.2em;
-  overflow: hidden;
-}
+/* Desktop size */
+@media (min-width: 768px) {
 
-.markers {
-  width: 25%;
-  font-size: 0.8em;
-  line-height: 1.2em;
-  float: left;
-}
+  .markers {
+    width: 25%;
+    font-size: 0.8em;
+    line-height: 1.2em;
+    float: left;
+  }
+
+  .WrapperBlock {
+    height: 100%;
+    width: 100%;
+    margin-bottom: 2em;
+    line-height: 1.2em;
+    overflow: hidden;
+  }
+
+  .speaker {
+    display: inline-block;
+    width: 50%;
+    padding-right: 2em;
+    vertical-align: middle;
+    color: color-mid-grey;
+    font-weight: bold;
+    text-transform: uppercase;
+    text-align: right;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
+
+  .text {
+    width: 75%;
+    font-size: 1em;
+    float: right;
+    clear: right;
+  }
 
-.text {
-  width: 75%;
-  font-size: 1em;
-  float: right;
-  clear: right;
 }
 
-.speaker {
-  display: inline-block;
-  width: 50%;
-  padding-right: 2em;
-  vertical-align: middle;
-  color: color-mid-grey;
-  font-weight: bold;
-  text-transform: uppercase;
-  text-align: right;
-  text-overflow: ellipsis;
-  overflow: hidden;
-  white-space: nowrap;
+/* Mobile devices */
+@media (max-width: 768px) {
+
+  .markers {
```

Comment:
In mobile view, the speaker labels and timecodes are just a _bit_ too close to the text. adding a `margin-bottom` would help

Reply:
ok changed to be 

```css
.markers {
...
    margin-bottom: 0.5em;
```
and
```css
.WrapperBlock {
...
    margin-bottom: 1em;
```

<img width="382" alt="screen shot 2019-02-28 at 19 45 01" src="https://user-images.githubusercontent.com/4661975/53593812-758d4500-3b91-11e9-995b-64294ca0fd31.png">

see what you think, we can tweak this